source/components/the-component-lifecycle.md

```diff
@@ -29,34 +141,32 @@ Ember guarantees that, by the time `didInsertElement()` is called:
    [`$()`][dollar]
    method.
 
-A component's [`$()`][dollar] method allows you to access the
-component's DOM element via jQuery. For example, you can set an
-attribute using jQuery's `attr()` method:
+A component's [`$()`][dollar] method allows you to access the component's DOM element via jQuery.
```

Comment:
Technically it returns a jQuery element, this is an occasional source of confusion so precise language is necessary.